### ISSUE TITLE
Schema.jsonify takes dump(...).data

### ISSUE THREAD

**Issue by jcassee** [marshmallow-code/flask-marshmallow#77]
Marshmallow (at least 3.0.0b8) has an implementation of `Schema.dump` that contains a dict of serialized data. However, flask-marshmallow's `Schema.jsonify` tries to take its `data` attribute. (See [source](https://github.com/marshmallow-code/flask-marshmallow/blob/7208b900ecdac881d272378c4ef1482741fe7b9b/flask_marshmallow/schema.py#L35).)

This seems to be a bug? It least in my code I get this exception:
```
[...]
  File "/[...]/flask_marshmallow/schema.py", line 35, in jsonify
    data = self.dump(obj, many=many).data
AttributeError: 'dict' object has no attribute 'data'
```

**Comment by sloria** [comment]
This issue should be fixed in 0.9.0

**Comment by jcassee** [comment]
Thanks, @sloria!
